src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -24,71 +24,82 @@ export plant_absorbed_ppfd,
+                        SW_IN:FT,
+                        ρ_leaf::FT,
                         LAI::FT,
                         K::FT,
                         _,
                         _,
+                        _,
     )
 
-Computes the absorbed photosynthetically active radiation in terms 
-of mol photons per m^2 per second (`APAR`).
+Computes the absorbed photon flux density in terms of mol photons per m^2 per 
+second for a radiation band. If the reflectance and radiation for NIR is passed, 
+computes ANIR and if PAR reflectance and rediation are passed, computes APAR.
 
-This applies the Beer-Lambert law, which is a function of photosynthetically
-active radiation (`PAR`; moles of photons/m^2/),
-PAR canopy reflectance (`ρ_leaf`), the extinction
-coefficient (`K`), leaf area index (`LAI`) and the clumping index (`Ω`). 
+This applies the Beer-Lambert law, which is a function of incident 
+radiation (`SW_IN`; moles of photons/m^2/), leaf reflectance
+(`ρ_leaf`), the extinction coefficient (`K`), leaf area index (`LAI`),
+and the clumping index (`Ω`). 
 The function takes in all parameters in the parameters struct for a 
-BeerLambertModel, along with the PAR, LAI, extinction coefficient K, and solar 
-zenith anlgle.
+BeerLambertModel, along with the SW_IN, LAI, extinction coefficient K, and solar 
+zenith angle.
 """
-function plant_absorbed_ppfd(
+function plant_absorbed_pfd(
     RT::BeerLambertModel{FT},
-    PAR::FT,
+    SW_IN::FT,
+    ρ_leaf::FT,
     LAI::FT,
     K::FT,
     _,
     _,
+    _,
 ) where {FT}
     RTP = RT.parameters
-    APAR = PAR * (1 - RTP.ρ_leaf) * (1 - exp(-K * LAI * RTP.Ω))
-    return APAR
+    AR = SW_IN * (1 - ρ_leaf) * (1 - exp(-K * LAI * RTP.Ω))
+    return AR
 end
 
 """
-    plant_absorbed_ppfd(RT::TwoStreamModel{FT},
+    plant_absorbed_pfd(RT::TwoStreamModel{FT},
+                        ρ_leaf,
+                        τ_leaf,
                         PAR::FT,
                         LAI::FT,
                         K::FT,
                         θs::FT,
+                        a_soil::FT,
     )
 
-Computes the absorbed photosynthetically active radiation in terms 
-of mol photons per m^2 per second (`APAR`).
+Computes the absorbed photon flux density in terms of mol photons per m^2 per 
+second for a radiation band. If the reflectance, radiation, transmittance, and 
+soil albedo for NIR is passed, computes ANIR and if PAR reflectance, rediation,
+transmittance, and soil albedo are passed, computes APAR.
 
 This applies the two-stream radiative transfer solution which takes into account
 the impacts of scattering within the canopy. The function takes in all 
 parameters from the parameter struct of a TwoStreamModel, along with the 
-incident PAR, LAI, extinction coefficient K, soil albedo from the 
+incident radiation, LAI, extinction coefficient K, soil albedo from the 
 canopy soil_driver, and solar zenith angle.
 """
-function plant_absorbed_ppfd(
+function plant_absorbed_pfd(
```

Comment:
ppfd

Reply:
Hmm, I changed to pfd (ppfd = photosynthetic flux density, and we are now using this for all flux densities, not just photosynthetic), but if this should be named differently let me know.

Reply:
sounds good! just a note to change here as well:https://github.com/CliMA/ClimaLSM.jl/blob/main/docs/src/APIs/canopy/RadiativeTransfer.md